src/components/LaunchDetailsPage/mocks/launchDetails.ts

```diff
@@ -0,0 +1,18 @@
+export const launchDetails = {
```

Comment:
provide type for this const

Reply:
added /types/ILaunchDetails.ts with interface